test/pj-rehearse-integration/expected_jobs.yaml

```diff
@@ -0,0 +1,398 @@
+- apiVersion: prow.k8s.io/v1
```

Comment:
How come `pj-rehearse` doesn't have the normal user-facing output?

Reply:
I was also wondering that. We could add a flag similar to `oc`'s `-o yaml` — which I imagine could be useful in other contexts ­— to avoid sacrificing the valuable default output, akin to what we did for `--debug-log`.

Reply:
It only has logging ATM... Maybe we could fix it but right now clear stdout is quite handy in dry-mode :)